index.html

```diff
@@ -60,6 +58,87 @@ <h2 class="title text-primary">I’m a software developer</h2>
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
+                        <div class="buttons">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
+                        </div>
+                        <div class="see-project">
+                            <button>See project</button>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="title about-me">
+                <h2 class="title">About Me</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="get-my-resume">
+                    <button>Get My Resume</button>
+                </div>
+            </section>
+                <section class="img-patholder language">
+                    <div class="gray-box">
+                        <h2 class="title">Languages</h2>                        
+                        <div class="buttons button-white-bg">
+                            <button type="button">ruby on rails</button>
+                            <button type="button">Css</button>
+                            <button type="button">javascript</button>
+                            <button type="button">html</button>
```

Comment:
- [ ] Each of these `.button`s should be inside a `li` tag. and the `div` for the `.buttons` class should be a `ul` instead. it would make more semantic sense.